Casks/amazon-ena-ethernet-dext.rb

```diff
@@ -0,0 +1,32 @@
+#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cask "amazon-ena-ethernet-dext" do
+    version "1.0.4"
+    dextDriverKitVersion = "21.2"
+  
+    if Hardware::CPU.arm?
+      if MacOS.version >= :monterey
```

Comment:
Question: what happens when these conditions aren't matched? An error?

Reply:
It looks like the install will try to continue. 
```
Error: Download failed on Cask 'amazon-ena-ethernet-dext' with message: undefined method `using' for nil:NilClass
```

I will add graceful failures.